pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -1121,39 +1131,26 @@ IdealState ensureAllPartitionsConsuming(TableConfig tableConfig, PartitionLevelS
           // 3. we should never end up with some replicas ONLINE and some OFFLINE.
```

Comment:
remove the "Resolved in " from point number 2, now that your PR handles this. I think that comment was making some assumptions that this will never happen after correcting the bug in that 2980 PR

Reply:
Ack